.github/workflows/ios-test.yml

```diff
@@ -10,6 +10,8 @@ jobs:
     runs-on: macos-latest
     steps:
       - uses: actions/checkout@v2
+        with:
+          lfs: true
```

Comment:
In iOS build job, screenshot files are not needed. So it's okey to remove this 👍🏼

Reply:
```suggestion
```